ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -47,17 +48,16 @@ class FaultInjector
     VelocityLimit=7, 
     NoForceData=8};
 
-  enum SystemFaults {
-    System=1, 
-    ArmGoalError=2, 
-    ArmExecutionError=4,
-    TaskGoalError=8, 
-    CamGoalError=16, 
-    CamExecutionError=32, 
-    PtGoalError=64, 
-    PtExecutionError=128, 
-    LanderExecutionError = 256};
-
+	static constexpr std::bitset<9> isSystem{		0b0'0000'0001 };
+	static constexpr std::bitset<9> isArmGoalError{		0b0'0000'0010 };
+	static constexpr std::bitset<9> isArmExecutionError{		0b0'0000'0100 };
+	static constexpr std::bitset<9> isTaskGoalError{	0b0'000'1000 };
```

Comment:
What is a Task Goal error?

Reply:
Similar to System, Task Goal Error is another category that JPL defined.